src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_product-cards/assets/106397016/4b517907-8fe6-4265-a98f-3af7bce8a42e)